README.md

```diff
@@ -0,0 +1,34 @@
+# Set-up a 'Hello-microverse' Project
+This is 'Hello-Microverse' project. The project is to help new students understand the basic requirements when designing a program. It helps new students get acquainted with microverse tools and workflows.   
+# Built with
+* Html
+* CSS
+* Git
+# Live Demo
+[Live Demo Link](http://127.0.0.1:5500/Hello-microverse/index.html)
```

Comment:
- [ OPTIONAL] Kindly remove this line because the project was not deployed. the port you're using is locally. only you can have access on it local. for example in my case to run your project the server I'm using is localhost:5500 instead of using live demo you can just say `N/A`